### DISCUSSION TITLE
Previewing/rendering images from relative path

### DISCUSSION THREAD

**Discussion by margarizaldi** [Alex-D/Trumbowyg#1347] [Q&A] [answered]
I have a very simple project which the admin page is in the different path with the frontend (no router, just directory based urls)

```
public/admin/index.php <-- here's the editor
public/index.php <-- the page to show the image
public/images/img-1.jpg <-- the image I want to show both in editor & frontend

```
I need the admin user able to manually type some relative link and see the preview on the editor...

`<img src='images/img-1.jpg />`

How to achieve that? Thanks.

**Comment by Alex-D** [comment, marked as answer]
Why not just use the absolute path?

`<img src='/images/img-1.jpg />`

**Comment by margarizaldi** [comment]
> Why not just use the absolute path?
> 
> `<img src='/images/img-1.jpg />`

Haven't tried but won't the editor be looking for `public/admin/images/img-1.jpg` instead of `public/images/img-1.jpg` ?

**Reply by Alex-D** [reply]
Nope, since there is a / before, it will be absolute, not relative

**Reply by margarizaldi** [reply]
Perfect... That really works... Thanks
